lib/handlers/base/api-handler.ts

```diff
@@ -187,10 +195,12 @@ export abstract class APIGLambdaHandler<
               ({ body, statusCode } = handleRequestResult);
             }
           } catch (err) {
-            log.error({ err }, 'Unexpected error in handler');
```

Comment:
seeing serialization issue here if err is `CustomError` with logging the error taking up to 650 ms